### PR TITLE
Add descriptions to swb and re-factor description, label handling

### PR DESCRIPTION
**Pull request by lokal-profil**
* Handle label, alias and description similarly in both swb and rbd
* Use new add_multiple methods in WikidataStuff
* Move json encoding of language dictionaries to separate function

Task: [T167018](https://phabricator.wikimedia.org/T167018)

### CODE REVIEW

File: WFD/RBD.py [hunk 1]
```diff
@@ -154,67 +156,60 @@ def create_new_rbd_item(self, entry_data, country):
...
         sometimes a comment.
 
-        The output is a dict with lang as key and a dict as value where that
-        dict in turn has a language and a value key. Both of these have string
-        values. Unless with_alias is provided.
+        The output is a dict with lang as key and a list of names as value.
 
         @todo Figure out how to handle internationalRBDName (which can be in
-              other languages, or a duplicate, or different english).
+              other languages, or a duplicate, or different English).
```
Comment (Vesihiisi) [inline WFD/RBD.py]: How is that presented -- does it indicate what language(s) it applies to?

File: WFD/WFDBase.py [hunk 2]
```diff
@@ -101,13 +104,19 @@ def commit_labels(self, labels, item):
...
-                                        caseSensitive=False)
+        if labels:
+            self.wd.add_multiple_label_or_alias(
+                labels, item, case_sensitive=False)
+
+    def commit_descriptions(self, descriptions, item):
+        """
+        Add description to item.
+
+        :param description: description object
```
Comment (Vesihiisi) [inline WFD/WFDBase.py]: :param descriptions:

File: WFD/WFDBase.py [hunk 3]
```diff
@@ -101,13 +104,19 @@ def commit_labels(self, labels, item):
...
-            for value in values:
-                self.wd.addLabelOrAlias(lang, value, item,
-                                        caseSensitive=False)
+        if labels:
+            self.wd.add_multiple_label_or_alias(
+                labels, item, case_sensitive=False)
+
+    def commit_descriptions(self, descriptions, item):
+        """
+        Add description to item.
```
Comment (Vesihiisi) [inline WFD/WFDBase.py]: descriptions, in plural, since we can add multiple

File: tests/test_wfdbot.py [hunk 4]
```diff
@@ -110,3 +110,89 @@ def test_validate_mapping_label(self):
...
+
+    """Test the convert_language_dict_to_json method."""
+
+    def test_convert_language_dict_to_json_empty(self):
+        self.assertEqual(
+            WfdBot.convert_language_dict_to_json({}, 'labels'),
+            {}
+        )
+
+    def test_convert_language_dict_to_json_singel(self):
```
Comment (Vesihiisi) [inline tests/test_wfdbot.py]: singel -> single

File: WFD/RBD.py [hunk 5]
```diff
@@ -154,67 +156,60 @@ def create_new_rbd_item(self, entry_data, country):
         :param entry_data: dict with the data for the rbd
         :param country: country code
         """
-        data = {
-            'labels': {},
-            'descriptions': {}}
-
-        data['labels'] = self.make_labels(entry_data)
-        data['descriptions'] = self.make_descriptions(entry_data, country)
+        labels = WfdBot.convert_language_dict_to_json(
```
Comment (Vesihiisi) [inline WFD/RBD.py]: What's the `country` parameter used for?

Comment (lokal-profil) [inline WFD/RBD.py]: It was used to set the country part of the description. This is now instead handled by the internal `self.country` (in `WfdBot`).

But I realised I need `rbd.set_common_values(data)` in order to set this.

Fixed

File: WFD/RBD.py (hunk 1 above)
Comment (lokal-profil) [inline WFD/RBD.py]: No it is simply a mess. Hence the todo =)

File: WFD/WFDBase.py (hunk 3 above)
Comment (lokal-profil) [inline WFD/WFDBase.py]: Fixed

File: WFD/WFDBase.py (hunk 2 above)
Comment (lokal-profil) [inline WFD/WFDBase.py]: Fixed

File: tests/test_wfdbot.py (hunk 4 above)
Comment (lokal-profil) [inline tests/test_wfdbot.py]: Fixed

File: WFD/RBD.py (hunk 5 above)
Comment (Vesihiisi) [inline WFD/RBD.py]: Then the parameter can be removed from the method definition here?

Comment (lokal-profil) [inline WFD/RBD.py]: Yes. (I've done a bunch of cleanup in #11 but forgot to add it here where it was introduced).

Fixed

**Pull request merged**